tests/test_plugin.py

```diff
@@ -81,3 +85,25 @@ def test_get_launch_attributes():
     expected_out = [{'value': 'Tag'}, {'key': 'Key', 'value': 'Value'}]
     out = get_launch_attributes(['Tag', 'Key:Value', ''])
     assert expected_out == out
+
+
+@mock.patch('pytest_reportportal.plugin.requests.get')
+def test_portal_on_maintenance(mocked_get, mocked_session):
+    """Test if portal on maintenance."""
+    mock_response = mock.Mock()
+    mock_response.text = "<title>Report Portal - Maintenance</title>"
```

Comment:
I'd use either compare lower case values of upper case ones.

Reply:
Checking "maintenance" word in msg.lower() now.